akka-http-core/src/test/scala/akka/http/scaladsl/model/headers/HeaderSpec.scala

```diff
@@ -23,7 +23,7 @@ class HeaderSpec extends FreeSpec with Matchers {
             CacheDirectives.`s-maxage`(1000)))
       }
       "failing parse run" in {
-        val Left(List(ErrorInfo(summary, detail))) = headers.`Last-Modified`.parseFromValueString("abc")
+        val Left(List(ErrorInfo(summary, detail, ""))) = headers.`Last-Modified`.parseFromValueString("abc")
```

Comment:
This original one would be how I expect it to be used by users if any users use it - the two param unapply - 
``case ErrorInfo(summary, detail) => ...` and that will break because of the binary and source incomp. introduced.

It may be a very rare use case though, I'm not sure.

Reply:
Thank you for pointing out.

I think that the possibility can not be denied as you wrote about such usecase.

I tried several methods in addition to try to solve incomp, but I could not solve it.
Maybe `unapply` generated by case class can not override and can it be defined in multiple.

Should we change ErrorInfo to normal class for to solve incomp？

Reply:
Changing it from a case class to a normal class (with unapply/copy methods) sounds fine to me!

Reply:
Yeah, maybe that is the only viable option

Reply:
Thanks for the replay.

I changed it and pushed it.

Reply:
Ok, thanks for fixing it :)